pkg/reconciler/kubernetes/tektonresult/transform.go

```diff
@@ -208,12 +225,33 @@ func updateApiConfig(p interface{}) mf.Transformer {
 				if !innerElem.IsValid() {
 					continue
 				}
-				if innerElem.Kind() == reflect.Bool {
+				switch innerElem.Kind() {
+				case reflect.Bool:
 					prop[ukey] = strconv.FormatBool(innerElem.Bool())
-				} else if innerElem.Kind() == reflect.Uint {
+					continue
+				case reflect.Int:
+					fallthrough
+				case reflect.Int8:
+					fallthrough
+				case reflect.Int16:
+					fallthrough
+				case reflect.Int32:
+					fallthrough
+				case reflect.Int64:
+					prop[ukey] = strconv.FormatInt(innerElem.Int(), 10)
+					continue
+				case reflect.Uint:
+					fallthrough
+				case reflect.Uint8:
+					fallthrough
+				case reflect.Uint16:
+					fallthrough
+				case reflect.Uint32:
+					fallthrough
+				case reflect.Uint64:
```

Comment:
@khrm why too many `fallthrough` use a single case.

Reply:
All these cases require the same operations. Without the fallthrough, our case won't go to that statement.

Reply:
@khrm I mean can change this to as follows,
```go
switch innerElem.Kind() {
case reflect.Bool:
	prop[ukey] = strconv.FormatBool(innerElem.Bool())
	continue

case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
	prop[ukey] = strconv.FormatInt(innerElem.Int(), 10)
	continue

case reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64:
	prop[ukey] = strconv.FormatUint(innerElem.Uint(), 10)
	continue
}
```